docs/example-embedded-web-server.md

```diff
@@ -0,0 +1,35 @@
+# Embedded web server examples
+
+The Java Buildpack can run applications which provide their own web server or servlet container, provided as JAR files.
+
+## Example
+
+This example uses Jetty as an embedded web server, but should be applicable for other technologies.
+
```

Comment:
Can you please update this example to use a [fenced code block](https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks).  You can get syntax highlighting if you append `java` to the opening blocks.